static/scss/homepage.scss

```diff
@@ -430,7 +452,7 @@ body.app-media {
 }
 
 .reasons-section {
-  background-color: $mm-brand-bg-dark;
+  background-color: $mm-brand-bg;
 
   .col-md-4 {
     padding: 0 3%;
```

Comment:
some of the changes in this file aren't going to pass the sass linter, it will probs be easier for you to just correct the formatting based on that